_posts/2016-09-00-edsls-part-1.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: It's programs all the way down
+
+meta:
+  nav: blog
+  author: adelbertc
+  pygments: true
+---
+
+*This is the first of a series of articles on "EDSLs in Scala."*
```

Comment:
An EDSL can be made of pure data, with no room for arbitrary transformations (i.e. `Functor`).  If the goal of this series is to focus on those for which lifting into `Free` is meaningful and the intent of the design, then this should be reflected; a series about EDSLs in general would also be interesting, but this article implies that the focus is on the aforementioned subset.